features/promotion/managing_catalog_promotions/creating_catalog_promotion.feature

```diff
@@ -44,4 +42,3 @@ Feature: Creating a catalog promotion
         Then there should be 1 new catalog promotion on the list
         And it should have "winter_sale" code and "Winter sale" name
         And the catalog promotion "Winter sale" should be available in channel "United States"
-        And this catalog promotion should be usable
```

Comment:
Why did you remove these steps?

Reply:
I have restored it only for scenario where CP has an action